packages/relay/src/lib/eth.ts

```diff
@@ -191,14 +199,25 @@ export class EthImpl implements Eth {
       if (blockCount <= 0) {
         return EthImpl.feeHistoryZeroBlockCountResponse;
       }
-
-      const cacheKey = `${constants.CACHE_KEY.FEE_HISTORY}_${blockCount}_${newestBlock}_${rewardPercentiles?.join('')}`;
-      let feeHistory: object | undefined = this.cache.get(cacheKey);
-      if (!feeHistory) {
-        feeHistory = await this.getFeeHistory(blockCount, newestBlockNumber, latestBlockNumber, rewardPercentiles, requestId);
-        if (newestBlock != EthImpl.blockLatest && newestBlock != EthImpl.blockPending) {
-          this.logger.trace(`${requestIdPrefix} caching ${cacheKey}:${JSON.stringify(feeHistory)} for ${constants.CACHE_TTL.ONE_HOUR} ms`);
-          this.cache.set(cacheKey, feeHistory);
+      let feeHistory: object | undefined;
+      if(this.getEthFeeHistoryFixedFee()) {
+        if(newestBlockNumber <= 0){
```

Comment:
Same here. The fact that you have `newestBlockNumber` means in some cases you made an extra call even though you don't need it in many cases

Reply:
Also when will `newestBlockNumber` ever be less than 0?

Reply:
Correct, I updated my comment above. 
`newestBlockNumber` would only have seen a mirror block call if the block is a number or its `latest` and in both cases it doesn't exist in the cache.

I think we should always attempt a call for `newestBlockNumber` with consideration for  caches.
No need to check if `latest` > `newestBlockNumber` right now cause honestly by the time a response is returned there's a new block with a fee, so we can assume this to be true in your PR

Reply:
I've improved the condition that checks for negative values, in reality the validation needs to be for the `oldestBlock` since they could input `newestBlock` positive, and oldest would still return a negative value due to the `blockCount` parameter. so I moved the validation around a bit and tests continued to pass as expected. 👍